fs/Kconfig

```diff
@@ -119,6 +119,13 @@ config FS_HEAPSIZE
 		Support for shm/tmpfs/fs_pseudofile.c ram based fs memory.
 		default 0 to use kmm directly. independent heap disabled
 
+config FS_REFCOUNT
+	bool "File reference count"
+	default !DEFAULT_SMALL
+	---help---
+		Enable will Records the number of filep references. The file is
+		actually closed when the count reaches 0
```

Comment:
my impression is that this kind of things should not be a user-visible option.

Reply:
In theory, it should be used by default and should not be configured by the user, but it actually increases the code size, so it is configurable.Any better suggestions? I can try to optimize